src/common/com/intellij/plugins/haxe/lang/psi/impl/AnonymousHaxeTypeImpl.java

```diff
@@ -39,9 +36,12 @@ public AnonymousHaxeTypeImpl(@NotNull ASTNode node) {
   @NotNull
   @Override
   public List<HaxeType> getHaxeExtendsList() {
```

Comment:
Nevermind.  I just checked a half-dozen or so, and they all are looping as one would hope.  Callers aren't expecting to do a list.getFirst() and be done.